src/main/java/ca/corefacility/bioinformatics/irida/model/sequenceFile/SequenceFile.java

```diff
@@ -123,6 +123,16 @@ public SequenceFile(Path sampleFile) {
 		this();
 		this.file = sampleFile;
 	}
+	
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public boolean isGzipped() {
+		String fileExtension = com.google.common.io.Files.getFileExtension(this.file.getFileName().toString());
```

Comment:
I was looking at this sort of thing instead of the file extension, but I'm not sure if it's worth the extra effort.  https://stackoverflow.com/questions/30507653/how-to-check-whether-file-is-gzip-or-not-in-java
 Particularly since this requires the file to be read and it may take longer than it's worth.  In general I think the file extension checking is fine but I wanted to discuss whether it's worth it.

Reply:
We actually already implemented this in the `GzipFileProcessor`, where we were checking for file magic numbers to see if it was gzipped. I moved this code to a `FileUtils` class, and am using this same code (instead of file extension) to check if the file is gzipped. Done in 051f2923711b2ad78e6408fe3ec6e182e0b70c8a